cime_config/testdefs/testlist_clm.xml

```diff
@@ -331,6 +331,24 @@
       <option name="comment"  >Science support for 1850 with clm5_1 for SP mode at f19</option>
     </options>
   </test>
+  <test name="SMS_Ld1" grid="f09_g17" compset="I1850Clm60Sp" testmods="clm/clm60cam60LndTuningMode">
+    <machines>
+      <machine name="derecho" compiler="intel" category="ctsm_sci"/>
+    </machines>
+    <options>
+      <option name="wallclock">00:20:00</option>
+      <option name="comment"  >Science support for 1850 with clm5_1 for SP mode at f09 with cam60</option>
```

Comment:
Oh, instances of clm5_1 should be changed to clm6_0.
Do a `git grep clm5_1` and see if it needs to be removed elsewhere...

Reply:
Got it.